test/ios_coverage_test.sh

```diff
@@ -131,6 +131,10 @@ EOF
   CFBundleShortVersionString = "1.0";
   CFBundleVersion = "1.0";
 }
+EOF
+
+  cat > app/coverage_manifest.txt <<EOF
+app/SharedLogic.m
```

Comment:
```suggestion
app/StandaloneTest.m
```

Reply:
`app/SharedLogic.m` should appear though in coverage right?

Reply:
hrm yea ok, worth a try, i didn't test after i fixed the core issues locally, so maybe this is fine as is